src/control/server/storage/config_test.go

```diff
@@ -1,5 +1,5 @@
 //
-// (C) Copyright 2019-2023 Intel Corporation.
+// (C) Copyright 2019-2024 Intel Corporation.
```

Comment:
You aim at introducing quite a number of new allowed configurations. I would expect a number of new tests here as well. Or am I missing something?